zk_toolbox/crates/zk_inception/src/commands/prover/init.rs

```diff
@@ -135,3 +132,27 @@ fn get_object_store_config(
 
     Ok(object_store)
 }
+
+fn init_bellman_cuda(shell: &Shell) -> anyhow::Result<()> {
```

Comment:
lets have it as an additional step

Reply:
What do you mean? Another command? Or separate download and installation?

Reply:
I mean could we add a separate command for `download and installation` only